tests/test_load.py

```diff
@@ -1749,7 +1749,7 @@ def test_resolve_trust_remote_code_future(trust_remote_code, expected):
 def test_reload_old_cache_from_2_15(tmp_path: Path):
     cache_dir = tmp_path / "test_reload_old_cache_from_2_15"
     builder_cache_dir = (
-        cache_dir / "polinaeterna___audiofolder_two_configs_in_metadata/v2-374bfde4f55442bc/0.0.0/7896925d64deea5d"
+        cache_dir / "polinaeterna___audiofolder_two_configs_in_metadata/v2-374bfde4f55442bc/0.0.0/cf191ad706de653e"
```

Comment:
@lhoestq note that the hash now will be different from before this PR. I hope this does not have a negative impact...

Reply:
ah it's not supposed to change if we want to support reloading old cache directories

Reply:
@lhoestq then we cannot change the builder code anymore?